README.md

```diff
@@ -17,22 +17,18 @@ the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/pub
 
 For more information, see [license](LICENSE.md).
 
-
 ## Deployment
 
-Refer to [docs.18f.gov](https://docs.18f.gov/getting-started/setup/) for getting set up with Cloud Foundry
+Refer to [docs.18f.gov](https://docs.18f.gov/getting-started/setup/) for getting
+set up with Cloud Foundry
 
+To push to production:
 
-To deploy:
-`cf target -o 18f -s dolores` 
+`cf push dolores-app`
 
-`cf set-env dolores HOST dolores-app.18f.gov`
-`cf set-env dolores APPLICATION_HOST dolores-app.18f.gov`
-`cf set-env dolores ASSET_HOST dolores-app.18f.gov`
-`cf set-env dolores DEFAULT_URL_HOST dolores-app.18f.gov`
+New migrations will be run automatically. See the [manifest](manifest.yml) for
+more details on the Cloud Foundry setup.
 
-To push to production:
-`cf push dolores`
-To push to staging
-`cf push dolores-staging`
+To set or change the value of an environment variable on production:
 
+`cf set-env dolores-app <ENVIRONMENT VARIABLE> dolores-app.18f.gov`
```

Comment:
One more small addition. Maybe say something like this after the last line:

To see existing environment variables on the app in production:

`cf env dolores-app`